Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -1672,6 +1729,19 @@ void function AddTurretSentry( entity turret )
 	turret.Minimap_AlwaysShow( TEAM_MILITIA, null )
 	turret.Minimap_SetHeightTracking( true )
 	turret.Minimap_SetCustomState( eMinimapObject_npc.FD_TURRET )
+	entity player = turret.GetBossPlayer()
+	file.players[ player ].deployedEntityThisRound.append( turret )
+	AddEntityDestroyedCallback( turret, FD_OnEntityDestroyed )
+}
+
+function FD_OnEntityDestroyed( ent )
```

Comment:
you need to set `untyped` for the file if you want to use this

Reply:
sry,I forget it.
Already add it as a new commit